docassemble/ALWeaver/data/questions/assembly_line.yml

```diff
@@ -238,26 +242,76 @@ help:
     There are ${len(all_fields)} fields total.
 
     `${all_fields}`
+
+---
+code: |
+  interview.country = 'US'
+  # This is weird but needed for check-in logic
+  # TODO(qs): adjust when we have a way to load in org defaults at runtime
+---
+event: update_state_list
+code: |
+  if action_argument('interview.country') and action_argument('interview.country') != interview.country:          
+    interview.country = action_argument('interview.country')
+    background_response('refresh')
+  background_response()    
 ---
 id: dependency question
-question: Which jurisdictions and organizations is this form a part of?
-subquestion: |
-  This will help the Weaver to add appropriate jurisdictions as dependencies for your interview. If you are unsure, leave the default options checked.
-help:
-  label: My jurisdiction or organization is not here
-  content: |
-    New jurisdictions will be added soon. You should add those dependencies manually in the Playground until then.
+question: |
+  Brand, Location and Identity
 fields:
-  - Jurisdictions: interview.jurisdiction_choices
+  - Author's name(s) (one per line): interview.author
+    datatype: area
+    rows: 3
+    # TODO(qs): update default if we have a way to load org defaults
+    default: |
+      % if user_logged_in():
+      ${ user_info().first_name } ${ user_info().last_name }
+      % else:
+      Court Forms Online
+      % endif
+    required: False
+    help: |
+      The information in this box will be public by default.
+      It will be used on GitHub and on the "About" page of the interview.
+  # TODO(qs): we could add email for github here too, but more complicated      
+  - note: |
+      ---
+  - Default country: interview.country
+    code: |
+      countries_list()
+    default: ${ interview.country }
+  - Default ${ safe_subdivision_type(interview.country) if safe_subdivision_type(interview.country) else 'State/Province'}: interview.state
+    code: |
+        states_list(country_code=interview.country) if pycountry.subdivisions.get(country_code=interview.country) else ()
+    default: ${ "MA" if interview.country == 'US' else '' }
+    required: False
+    
+  - note: |
+      ---
+  - Jurisdiction package: interview.jurisdiction_choices
```

Comment:
I think the brand distinction you made above is very helpful. Here the idea is to let people include a package that someone else maintains which provides branding-neutral information that is specific to any jurisdiction. It does not need to be a state--it could be a country, a city, etc.

I don't think this is restricted to courts. E.g., we might put a glossary of terms that are state or country-specific in a package like this.